actions/image_partition_action.go

```diff
@@ -479,7 +479,17 @@ func (i ImagePartitionAction) Run(context *debos.DebosContext) error {
 		if i.PartitionType == "gpt" {
 			name = p.PartLabel
 		} else {
-			name = "primary"
+			if cap(i.Partitions) <= 4 {
```

Comment:
```suggestion
			if len(i.Partitions) <= 4 {
```
cap is the capacity of the  array; the actual length might be shorter